API-strategie-extensies/ext-geospatial.md

```diff
@@ -123,18 +123,48 @@ The guiding priciples for CRS support:
-  <p>The coordinate reference system (CRS) for both the request and the response are passed as part of the request headers and reponse headers. In case this header is missing, send the HTTP status code <code>412 Precondition Failed</code>.</p>
+  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request and the desired CRS of the response as parameters</p>
+  <p>The coordinate reference system (CRS) for both the request and the response are passed as parameters. The CRS of the geometry in the response is specified using a header.</p>
 </div>
 
-The following headers are purely meant for negotiation between the client and the server. Depending on the application, the request not only contains geometries but also specific meta data, e.g. the original realistion including the collection date.
+This method conforms to the standard OGC API Features part 2: Coordinate Reference Systems by Reference. 
+
+Use the following parameters: 
+- A parameter `crs` to specify the CRS used in a request. The value of the parameter is a URI identifier of the CRS. If the `crs` parameter is specified, then the coordinates of all geometry-valued properties in the response document must be presented in the requested CRS. If the requested CRS is not available, respond with HTTP status code 400.
+- A parameter `bbox-crs` to assert the CRS used for the coordinate values, in case the `bbox` parameter is used in the request. 
+<!-- Deze is nieuw, extra toelichting nodig als we die opnemen. -->
+
+The CRS of the geometry in the response is specified using the header `Content-Crs`. This mechanism allows a server to clearly and unambiguously assert the CRS and axis order being used in a response document independent of the requested output format, which is important because of the inconsistent provision of CRS metadata in geospatial encodings and the continued confusion caused by the axis order of coordinates.
+
+Use the following URIs to specify the CRS:
+
+|CRS|URI|
+|-|-|
+|ETRS89, 2D, European|http://www.opengis.net/def/crs/EPSG/9.9.1/4258|
+|ETRS89, 3D, European|http://www.opengis.net/def/crs/EPSG/9.9.1/4937|
+|WGS84, global|http://www.opengis.net/def/crs/OGC/1.3/CRS84|
+|Web Mercator, global|http://www.opengis.net/def/crs/EPSG/9.9.1/3857|
+|RD/Amersfoort, 2D, Dutch|http://www.opengis.net/def/crs/EPSG/9.9.1/28992|
+|RD/Amersfoort + NAP, 3D, Dutch|http://www.opengis.net/def/crs/EPSG/9.9.1/7415|
+
+<!-- Deze URIs moeten gecheckt worden -->
+
+For backwards compatibility, an older method of specifying CRS in the headers of requests is retained as a deprecated method. APIs that already support the (deprecated) header method can add support for the parameter method while still supporting the header method for a certain period.  Supporting both the new method (using parameters) and the old (using headers) is trivial. 
```

Comment:
Waarschijnlijk een uitzonderlijke situatie, maar wat als een `client` zowel de parameter als de header gebruikt in het verzoek? Eisen we dan dat de parameter versie altijd voorrang heeft, aangezien we daar naar toe willen?

Dit kan alleen optreden als de parameter en header verschillende CRS-en vragen, maar ik denk dat het goed is om hier een tekst over toe te voegen.

Reply:
over het algemeen overschrijft een parameter in de url een parameter in de header, denk bijvoorbeeld aan language, de browser stuurt de browser taal, als je echter een andere taal wilt, kun je dit eventueel overschrijven via de corresponderende url parameter, ik weet niet of dit ergens officieel beschreven is, maar is wat ik vaak zie bij api's

Reply:
Ik voeg een note toe die het gedrag beschrijft zoals @pvgenuchten hier voorstelt.